spec/dummy/config/database.yml

```diff
@@ -6,6 +6,7 @@ common: &common
   template: template0
   pool: 5
   timeout: 5000
+  port: <%= ENV['POSTGRES_PORT'] || '34199' %>
```

Comment:
I'm making the local `docker-compose.yml` start a postgres server on port `34199` rather than the default 5432.  This allows devs to work on partitioned while `jobseekerltd/localhost` is running.